packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/developer-settings-organisation-tab.tsx

```diff
@@ -1,16 +1,34 @@
 import * as React from 'react'
-import { FlexContainerResponsive, Content, FlexContainerBasic } from '@reapit/elements'
+import { FlexContainerResponsive, Content, FlexContainerBasic, Button, LevelRight } from '@reapit/elements'
 import { Tabs } from '../tabs'
+import DeveloperInviteModal from '@/components/ui/developer-invite-member-modal'
+
+export const handleToggleVisibleModal = (setIsInviteModalOpen: React.Dispatch<boolean>, isVisible: boolean) => () =>
+  setIsInviteModalOpen(isVisible)
 
 const DevelperSettingsOrganisationTabPage: React.FC = () => {
+  const [isInviteModalOpen, setIsInviteModalOpen] = React.useState<boolean>(false)
+
   return (
-    <FlexContainerBasic flexColumn hasPadding>
-      <Content>
-        <FlexContainerResponsive flexColumn hasBackground hasPadding>
-          <Tabs />
-        </FlexContainerResponsive>
-      </Content>
-    </FlexContainerBasic>
+    <>
+      <FlexContainerBasic flexColumn hasPadding>
+        <Content>
+          <FlexContainerResponsive flexColumn hasBackground hasPadding>
+            <Tabs />
+            <LevelRight>
+              <Button type="button" variant="primary" onClick={handleToggleVisibleModal(setIsInviteModalOpen, true)}>
+                Invite New Member
+              </Button>
+            </LevelRight>
+          </FlexContainerResponsive>
+        </Content>
+      </FlexContainerBasic>
+
+      <DeveloperInviteModal
```

Comment:
I think should unmount modal when you don't use it
`isInviteModalOpen && <DeveloperInviteModal ....`

Reply:
The unmount logic is already in the Modal itself
<img width="735" alt="Screen Shot 2020-06-05 at 11 35 15" src="https://user-images.githubusercontent.com/26746668/83837464-b3780200-a720-11ea-9ff2-096573af6d28.png">

Reply:
ok